integration-test/Analysis/LicenseScannerSpec.hs

```diff
@@ -79,3 +80,19 @@ spec = do
             mitUnit = fromMaybe emptyLicenseUnit (head' $ NE.filter (\u -> licenseUnitName u == "mit") us)
             apacheUnit :: LicenseUnit
             apacheUnit = fromMaybe emptyLicenseUnit (head' $ NE.filter (\u -> licenseUnitName u == "apache-2.0") us)
+
+    it "should filter licenses if LicenseScanPathFilters is set" $ do
+      extractedDir <- getArtifact recursiveArchive
```

Comment:
This will not pass until https://github.com/fossas/basis/pull/1774 is merged, a release is cut, and we update the themis release tag in `vendor_download.sh`

Reply:
They pass now